### ISSUE TITLE
CSS fixes to docked UI sizing

### ISSUE THREAD

**Issue by DC23** [DC23/jd-easytimekeeping#305]
@GregoryWarn 
I've been looking closer at the CSS I'm using for the width of the docked UI form. I think I can improve it. At the moment, if you are showing the long-form time string then it will expand the form to fit the string but it will never shrink again. 

One option is to keep the UI fixed width but allow the string content to wrap to the next line if needed. The other option will be single line, but to expand and shrink the UI dynamically.

I need to learn more CSS!

**Comment by DC23** [comment]
I think I've improved the situation. Long text now wraps to extra lines for both the floating and docked UI. I added the same CSS to both the time and the fuzzy time.

![Screenshot 2024-12-17 144903](https://github.com/user-attachments/assets/d8f0c208-41f4-43e8-9575-eb6c5143e7eb)
![Screenshot 2024-12-17 145152](https://github.com/user-attachments/assets/04b3bbee-65cf-49fa-99da-d031ccaac66f)

![Screenshot 2024-12-17 145746](https://github.com/user-attachments/assets/366a80af-df41-4219-9b41-545b928a922e)

There might still be issues, but it's an improvement.
